hnn_core/drives.py

```diff
@@ -302,115 +302,6 @@ def _drive_cell_event_times(drive_type, dynamics, tstop, target_type='any',
     return event_times
 
 
-def drive_event_times(drive_type, target_cell_type, params, gid, trial_idx=0):
```

Comment:
oh dear! How did we forget to remove this?! Good catch!

Reply:
This was a wonderful relic of the past that I think you had originally flagged for removal. Hopefully this will make our tests less confusing!